intro/webhooks_intro.md

```diff
@@ -193,27 +195,30 @@ The *Status* of the registration should show as **Active**. If the registration
 
 ![Event Registration Details tab in Adobe Developer Console](../img/events-registration-details.png)
 
-### Troubleshooting a Disabled Registration Status
+### Troubleshooting Unstable/Disabled Registration Status
```

Comment:
Should we provide details for this state transition - When does an event-registration go from "Unstable or Disabled" to "Active/Stable/Healthy"?

Reply:
Do you think it will add value? If I think from a developer perspective, I would be concerned as to why my webhook registration is marked as `Unstable`/`Disabled` (which is covered in the faq doc). If it shows up as `Active` then I may not worry what the previous state was.

Also, transition from a `Disabled to Active` state requires explicit action from the developer (which is covered in the faq doc). However, transition from `Unstable` to `Active` state is implicit as long as the webhook becomes reachable again.

I would like to keep things simple but if you feel otherwise, let me know.

Reply:
The only transition left unanswered is from `Unstable` to `Active`. See if this looks good to you - https://github.com/AdobeDocs/adobeio-events/pull/87/files#r748027235. If not, then we can leave it as is.

Reply:
I added that. See https://github.com/AdobeDocs/adobeio-events/pull/87/commits/060427dec8a16358a026713555f8e15a9a386e82.